src/SWP/Bundle/ContentBundle/Model/RouteToArticleInterface.php

```diff
@@ -0,0 +1,67 @@
+<?php
+
+/**
+ * This file is part of the Superdesk Web Publisher Content Bundle.
+ *
+ * Copyright 2016 Sourcefabric z.ú. and contributors.
+ *
+ * For the full copyright and license information, please see the
+ * AUTHORS and LICENSE files distributed with this source code.
+ *
+ * @copyright 2016 Sourcefabric z.ú.
+ * @license http://www.superdesk.org/license
+ */
+namespace SWP\Bundle\ContentBundle\Model;
+
+use SWP\Component\Storage\Model\PersistableInterface;
+
+interface RouteToArticleInterface extends PersistableInterface
```

Comment:
I've added get and set rule. I assume we don't want get and set tenant code in here.